hopglass-server.js

```diff
@@ -480,9 +480,24 @@ function getFfmapJson(stream) {
 
 function getMetrics(stream) {
   getData()
-  var save = (n, id, stream, what, where) => {
-    if (_.has(n, what))
-      stream.write((where ? where : what.replace(/\./g, '_')) + id + ' ' +  _.get(n, what) + '\n')
+  var save = (n, stream, labels, what, value) => {
```

Comment:
Well, "what" is actually exactly the same as it was originally, which is what I'd call the metrics key from the statistics json. It is also used to form the prometheus metric name. 

And "value" is its value which is to be the value of the prometheus metric.

Also I tried to conserve the original functionality of the "save" function in a way that "what" can be used as both:
- as a key to the statistics dict in order to extract a value
- as a prometheus metrics name
the former being overruled by specifying a an optional value argument to the save function.